pkg/integrations/prometheus/promql/metricsSearchHandler.go

```diff
@@ -479,6 +479,12 @@ func ProcessGetMetricTimeSeriesRequest(ctx *fasthttp.RequestCtx, myid uint64) {
 	ctx.SetStatusCode(fasthttp.StatusOK)
 }
 
+func ProcessGetMetricFunctionsRequest(ctx *fasthttp.RequestCtx, myid uint64) {
+	jsonData := `[{"fn": "abs", "name": "Absolute", "desc": "Returns the absolute value of a metric.", "eg": "abs(avg (system.disk.used{*}))"}, {"fn": "log2", "name": "Logarithm base 2", "desc": "Returns the base-2 logarithm of a metric.", "eg": "log2(avg (system.disk.used{*}))"}]`
```

Comment:
nit: split `jsonData` across multiple lines so it's easier to read (especially as we add more functions here)